client/my-sites/themes/theme-options.js

```diff
@@ -49,9 +48,31 @@ const purchase = config.isEnabled( 'upgrades/checkout' )
 		} ),
 		getUrl: getThemePurchaseUrl,
 		hideForTheme: ( state, themeId, siteId ) => (
+			isJetpackSite( state, siteId ) || // No individual theme purchase on a JP site
+			! getCurrentUser( state ) || // Not logged in
+			! isThemePremium( state, themeId ) || // Not a premium theme
+			isPremiumThemeAvailable( state, themeId, siteId ) || // Already purchased individually, or thru a plan
+			isThemeActive( state, themeId, siteId ) // Already active
+		)
+	}
+	: {};
+
+const upgradePlan = config.isEnabled( 'upgrades/checkout' )
+	? {
+		label: i18n.translate( 'Upgrade to activate', {
+			comment: 'label prompting user to upgrade the Jetpack plan to activate a certain theme'
+		} ),
+		extendedLabel: i18n.translate( 'Upgrade to activate', {
+			comment: 'label prompting user to upgrade the Jetpack plan to activate a certain theme'
+		} ),
+		header: i18n.translate( 'Upgrade on:', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **41** times:
`translate( 'Upgrade', { context: 'verb'} )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.